src/astro/utils/table_handler.py

```diff
@@ -1,13 +1,25 @@
 import inspect
-from typing import Optional
+from abc import ABC
+from typing import Callable, Optional, Tuple
 
-import pandas
+from astro.databases import create_database
+from astro.sql.table import Metadata, Table
 
-from astro.settings import SCHEMA
-from astro.sql.table import Table
 
+class TableHandler(ABC):
+    """This class contains any functions involving modifying or reading from a Table object"""
+
```

Comment:
Do we use this class anywhere besides SqlDecorator? If we are not using it anywhere but with SqlDecorator, can we merge it?